detox/src/ios/expect.js

```diff
@@ -284,6 +291,9 @@ class Element {
     this._selectElementWithMatcher(this._originalMatcher._avoidProblematicReactNativeElements());
     return await new ActionInteraction(this, new SwipeAction(direction, speed, percentage)).execute();
   }
+  async scrollDatePickerAtColumnToValue(column,value) {
```

Comment:
This might need a name change as well, so we won't get confused by arg order.
`setPickerColumnToValue` or something similar.